### PR TITLE
Reword indexed updates metrics

### PR DESCRIPTION
**Pull request by nicu-da**

### CODE REVIEW

File: ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala [hunk 1]
```diff
@@ -21,8 +21,9 @@ class IndexedUpdatesMetrics(prefix: MetricName, metricFactory: MetricsFactory) {
   )
 
   @MetricDoc.Tag(
-    summary = "Updates processed by the indexer",
-    description = "Represents the total number of updates processed, that are sent for indexing.",
+    summary = "Number of transactions processed.",
+    description =
+      "Represents the total number of transaction acceptance, transaction rejection, package upload, party allocation, etc. processed.",
```
Comment (miklos-da) [inline ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala:26]:
```suggestion
      "Represents the total number of transaction acceptance, transaction rejection, package upload, party allocation, etc. events processed.",
```

File: ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala [hunk 2]
```diff
@@ -10,7 +10,7 @@ import com.daml.metrics.api.{MetricDoc, MetricHandle, MetricName}
 class IndexedUpdatesMetrics(prefix: MetricName, metricFactory: MetricsFactory) {
 
   @MetricDoc.Tag(
-    summary = "Number of events that will be metered",
+    summary = "Number of ledger events that are metered",
```
Comment (miklos-da) [inline ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala:13]: For consistency (we seem to enter full sentences as `summary`):
```suggestion
    summary = "Number of ledger events that are metered.",
```

File: ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala [hunk 3]
```diff
@@ -21,8 +21,9 @@ class IndexedUpdatesMetrics(prefix: MetricName, metricFactory: MetricsFactory) {
...
 
   @MetricDoc.Tag(
-    summary = "Updates processed by the indexer",
-    description = "Represents the total number of updates processed, that are sent for indexing.",
+    summary = "Number of transactions processed.",
+    description =
+      "Represents the total number of transaction acceptance, transaction rejection, package upload, party allocation, etc. processed.",
     qualification = Debug,
   )
   val eventsMeter: MetricHandle.Meter =
```
Comment (miklos-da) [inline ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala:29]: I was wondering if we wanted to align the Prometheus descriptions with auto-generated summaries (i.e., `MetricDoc.Tag` annotations) in the metrics doc?

**Review by miklos-da — APPROVED** [review]

File: ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala (hunk 3 above)
Comment (nicu-da) [reply ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala:29]: Yes, good point.

**Pull request merged**